packages/amplify-graphql-transformer-core/src/transformation/transform.ts

```diff
@@ -250,9 +264,113 @@ export class GraphQLTransform {
       reverseThroughTransformers -= 1;
     }
     this.collectResolvers(context, context.api);
+    if (this.overrideConfig?.overrideFlag) {
+      this.applyOverride(stackManager);
+      return this.synthesize(context);
+    }
     return this.synthesize(context);
   }
 
+  private applyOverride = (stackManager: StackManager) => {
+    let stacks: string[] = [];
+    let amplifyApiObj: any = {};
+    stackManager.rootStack.node.findAll().forEach(node => {
+      const resource = node as CfnResource;
+      if (resource.cfnResourceType === 'AWS::CloudFormation::Stack') {
+        stacks.push(node.node.id.split('.')[0]);
+      }
+    });
+
+    stackManager.rootStack.node.findAll().forEach(node => {
+      const resource = node as CfnResource;
+      let pathArr;
+      if (node.node.id === 'Resource') {
+        pathArr = node.node.path.split('/').filter(key => key !== node.node.id);
+      } else {
+        pathArr = node.node.path.split('/');
+      }
+      let constructPathObj: ConstructResourceMeta;
+      if (resource.cfnResourceType) {
+        // console.log('path',node.node.path);
+        // console.log('path1',pathArr);
+        constructPathObj = getStackMeta(pathArr, node.node.id, stacks, resource);
+        // console.log(constructPathObj);
+        if (!_.isEmpty(constructPathObj.rootStack)) {
+          // api scope
+          const field = constructPathObj.rootStack!.stackType;
+          const resourceName = constructPathObj.resourceName;
+          if (!amplifyApiObj[field]) {
+            amplifyApiObj[field] = {};
+          }
+          amplifyApiObj[field][resourceName] = resource;
+        } else if (!_.isEmpty(constructPathObj.nestedStack)) {
+          const fieldType = constructPathObj.nestedStack!.stackType;
+          const fieldName = constructPathObj.nestedStack!.stackName;
+          const resourceName = constructPathObj.resourceName;
+          if (constructPathObj.resourceType.includes('Resolver')) {
+            // console.log("constrcut",constructPathObj);
+            if (amplifyApiObj[fieldType][fieldName]['resolvers']) {
+              amplifyApiObj[fieldType][fieldName]['resolvers'][resourceName] = resource;
+            } else {
+              amplifyApiObj[fieldType][fieldName]['resolvers'] = {};
+              amplifyApiObj[fieldType][fieldName]['resolvers'][resourceName] = resource;
```

Comment:
use [`_.set()`](https://lodash.com/docs/4.17.15#set) to avoid all of this mess

Reply:
I will attend this when merging to master . Need to get the tagged release off